worlds/doom_1993/Options.py

```diff
@@ -80,7 +85,8 @@ class Episode3(DefaultOnToggle):
     "random_monsters": RandomMonsters,
     "random_pickups": RandomPickups,
     "allow_death_logic": AllowDeathLogic,
-    "death_link": DeathLink,
+    "start_with_computer_area_maps": StartWithComputerAreaMaps,
+    "DeathLink": DeathLink,
```

Comment:
What is the motivation to changing the yaml option name for death link to not use the same convention as other options?

Reply:
A suggestion then, have the yaml option be "death_link" still, but change the key in your `fill_slot_data` to "DeathLink". That way it can stay consistent in yaml but APCpp can see the expected key.

Reply:
Ok, that's what I've done in the previous PR to change the key in init.py, but was told to change the library instead.
For .net, really? I looked at the source to make sure its consistent. It looks like it's looking for "DeathLink":
```
        private static void EnsureDeathLinkTagIsSet(ArchipelagoSession session)
        {
            if (!session.Tags.Contains("DeathLink"))
```
But I might misunderstand the C# code as its not one of my strong languages.

Then if client libs are supposed to support both, I'll add back the check in APCpp by noobykind, and even PR a change in it. "in the future, we prefer this syntax"

Reply:
thats the tag not the slot data.

Reply:
But that's all client side handled. Generation doesn't need to worry about tags.

Reply:
I've put back `death_link`. The client library still supported both (I modified it a while ago). So just pretend you didn't see that.